src/tests/simulation/src/slt.rs

```diff
@@ -48,6 +48,10 @@ pub async fn run_slt_task(cluster: Arc<Cluster>, glob: &str, opts: &KillOpts) {
             // Simply ignore the tpch test cases when enable kill nodes.
             continue;
         }
+        if kill && path.ends_with("visibility_all.slt") {
```

Comment:
These test cases are skipped in recovery test because there is no guarantee for data persistency for them when `flush(checkpoint=false)`.